index.html

```diff
@@ -103,7 +103,7 @@
 									<li><a target="_self" class="inner-link" href="#subscribe">Subscribe</a></li>
 									<li><a target="_self" href="http://blog.fossasia.org">Blog</a></li>
 									<li class="has-dropdown"><a href="http://events.fossasia.org">Events</a>
-										<ul class="nav-dropdown events" style="min-height: 570px"> <!--whenever you add/remove an option, change the style="min-height:425px" by +- 25px-->
+										<ul class="nav-dropdown events" style="min-height: 559px"> <!--whenever you add/remove an option, change the style="min-height:425px" by +- 25px-->
```

Comment:
you need to increase or decrease by 25px

Reply:
Okay will decrease it by 25px